openedx/features/badging/tests/test_handlers.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
+from edx_notifications.exceptions import ItemNotFoundError
+from edx_notifications.lib.publisher import get_notification_type
+
+from openedx.features.badging.constants import EARNED_BADGE_NOTIFICATION_TYPE, JSON_NOTIFICATION_RENDERER
+
+from .. import handlers as badge_handlers
+
+
+class BadgeHandlerTestCase(TestCase):
+
+    def test_register_notification_types(self):
+        """
+        Test that notification type is registered correctly
+        :return: None
+        """
+        # register badge notification type
+        badge_handlers.register_notification_types(None)
+
+        badge_notification_type = get_notification_type(EARNED_BADGE_NOTIFICATION_TYPE)
+
+        self.assertEqual(badge_notification_type.name, EARNED_BADGE_NOTIFICATION_TYPE)
```

Comment:
This seems redundant, since we are getting notification type by name and then asserting on its name.

What about if we check `badge_notification_type` is not `None`.

Reply:
it never returns None instead throw an exception if type does not exists.So putting a check on None wouldn't be helpful.